test/service.ts

```diff
@@ -41,8 +39,11 @@ const fakeUtil = extend({}, util, {
   }
 });
 
-function FakeService() {
-  this.calledWith_ = arguments;
+class FakeService {
+  calledWith_: IArguments;
+  constructor() {
```

Comment:
Hopefully sometime in the future: `constructor(...args) { this.calledWith_ = args; }`